### PR TITLE
Simplify Logger

### PR DESCRIPTION
**Pull request by adrian-javid**
* Replaced all `Logger::WriteFormattedLine` with `Logger::WriteLine`.
* Removed `Logger::WriteFormattedLine`.

### CODE REVIEW

File: sandman/source/schedule.cpp [hunk 1]
```diff
@@ -223,10 +223,23 @@ static void ScheduleLogLoaded()
 			"up" : "down";
 			
 		// Print the event.
-		Logger::WriteFormattedLine("\t+%01ih %02im %02is -> %s, %s", delayHours, delayMin, 
-			delaySec, event.m_controlAction.m_controlName, actionText);
+		Logger::WriteLine(/* Set fill to `0`. */
```
Comment (shawn-lindberg) [inline sandman/source/schedule.cpp:226]: This comment is superfluous.

**Review by shawn-lindberg — CHANGES_REQUESTED** [review]
Overall looks good, just two minor changes requested.

File: sandman/source/reports.cpp [hunk 2]
```diff
@@ -357,8 +356,9 @@ void ReportsAddControlItem(std::string const& controlName, Control::Actions cons
 {
 	if ((action < 0) || (action >= Control::kNumActions))
 	{
-		Logger::WriteFormattedLine("Could not add control item to the report because it contains an invalid "
-			"action %d!", action);
+		Logger::WriteLine(
```
[the remark this answers is not in the document]
Comment (adrian-javid) [reply sandman/source/reports.cpp:359]: The string is too large to fit in 100 characters here. I'll break it into two quoted sections.

[the remark this answers is not in the document]
Comment (adrian-javid) [reply sandman/source/reports.cpp]: Resolved in 8d6ef78af5070176dd1b784d759b8d47ae45ac38.

File: sandman/source/schedule.cpp (hunk 1 above)
Comment (adrian-javid) [reply sandman/source/schedule.cpp]: I removed the comment that says "``/* Set fill to `0`. */``" in 1b6256e475da279e68dbb1eee4df2a6b25a0170e.

**Review by shawn-lindberg — APPROVED** [review]

**Pull request merged**